.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Author, Reviewer, and Assignee of a PR
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
+      A Reviewer reviews the change of a PR,
```

Comment:
If I remove that comma, that line will change from
> reviews the change of a PR, leaves comments

to

> reviews the change of a PR leaves comments

That doesn't seem right, does it?